### ISSUE TITLE
This tag "Yuuna" keeps appearing on my cards?

### ISSUE THREAD

**Issue by ONEsuzuki** [lrorpilla/jidoujisho#347]
![image](https://github.com/lrorpilla/jidoujisho/assets/111155695/6ac64806-444e-4a72-9949-727a0ad25996)
_exported tags on one of my anki cards_
I'm not quite sure how to remove this tag, I don't even have tags on my export enabled and it still exports this tag, why? 
Appreciate the help!

**Comment by brishtibheja** [comment]
Isn't this just default? You can't remove it although you may edit the cards later. I personally would rather keep it in case it helps me in a search.

btw if you don't know the dev is in a hiatus for now.

**Comment by ONEsuzuki** [comment]
@brishtibheja Thanks for the head up on the dev, I do see where you're coming from especially if you're using the apps default card template, there definitely should be an option to remove it though, it's annoying having to remove the tag every time even with anki shortcuts.

**Comment by brishtibheja** [comment]
yeah I can understand. I actually import my yomichan cards to the same note type so that's my reasoning. but actually I started using yomichan only recently, tbh I just never look at tags unless I'm searching for something.

but actually, albeit very slowly, irorpilla is working on the new version so there will be some news this year. oh, and another person is working to integrate https://reader.mokuro.app so we'll see some new developments.

**Comment by ONEsuzuki** [comment]
Didn't even realize that Mokuro developed an app I remember downloading it with python along time ago, I also read somewhere about importing whole manga playlists or have it recognise or bulk convert as a question in the issues, lets hope that's the plan, looking forward to the next release. 
Have a nice day.

**Comment by brishtibheja** [comment]
I would just add another thing in case you need it. You can find pre-processed manga in https://www.mokuro.moe/manga/

People are still uploading in TMW server's mokuro thread. So if you like manga you can just have them from here! じゃね〜

**Comment by ONEsuzuki** [comment]
@brishtibheja  
Massive THANKS! For showing me https://www.mokuro.moe/manga/
